src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,48 @@
+package racingcar.model;
+
+import static racingcar.utlis.RandomNumberGenerator.*;
+
+import racingcar.validator.NameValidator;
+
+public class Car {
+
+	static final int MOVE_CRITERIA = 4;
```

Comment:
전체적으로 접근제어자를 기본으로 private으로 설정하고 필요에 따라 바꿔주세요.
접근제어자도 신경을 써주시면 좋을 것 같아요.

```suggestion
	private static final int MOVE_CRITERIA = 4;
	
```